app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Posts</h1>
\ No newline at end of file
+<div class="posts">
+ <%= render "shared/details", user: @user %>
+  <div>
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+       <%= render "shared/posts_details", post: post %>
+        <div class="comments">
+        <% if post.comments.empty? %>
+          <p>Empty</p>
+        <% else %>
+            <% post.comments.each do |comment| %>
+             <p class="comment-body"><%= comment.author.name %>: <%= comment.text %></p>
+            <%end%>
```

Comment:
- [ ] Kindly make sure to use the `recent_comments` method to render the 5 most recent comments in the user posts index page, for now, your rendering all the comments